tests/Webcreate/Vcs/Git/Parser/CliParserTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Test\Webcreate\Vcs\Git\Parser;
+
+use Webcreate\Vcs\Git\Parser\CliParser;
+
+class CliParserTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var CliParser
+     */
+    private $parser;
+
+    public function setUp()
+    {
+        $client = $this->prophesize('Webcreate\Vcs\Git');
+
+        $this->parser = new CliParser();
+        $this->parser->setClient(
+            $client->reveal()
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_parses_diff_output()
+    {
+        $diffOutput = <<<EOT
+A       tests/Webcreate/Vcs/Git/Parser/CliParserTest.php
+R       Webcreate/Vcs/Svn/WorkingCopy.php
+R062    Webcreate/Vcs/Svn/SvnAdmin.php
```

Comment:
The syntax is not correct here. Git returns also the final file path on this line. For example:

```
M       virtualization/salt/pillar/prod.sls
M       virtualization/salt/salt/roles/app/init.sls
M       virtualization/salt/salt/roles/elk/files/logstash.conf
R062    virtualization/salt/salt/roles/app/files/filebeat.yml   virtualization/salt/salt/roles/filebeat/files/filebeat.jinja
A       virtualization/salt/salt/roles/filebeat/init.sls
```